### PR TITLE
Support addStyleTag and addScriptTag

### PR DESCRIPTION
**Pull request by paresharma**
This PR just supports for addStyleTag and addScriptTag which I needed for a recent project.

Would appreciate if this could be merged upstream and released as a patch or part of next release. Open to any feedback or changes needed.

### CODE REVIEW

File: lib/grover/js/processor.js [hunk 1]
```diff
@@ -101,6 +101,18 @@ const _processPage = (async (convertAction, urlOrHtml, options) => {
       }
     }
 
+    // add styles (if provided)
+    const addStyleTag = options.addStyleTag; delete options.addStyleTag;
```
Comment (abrom) [inline lib/grover/js/processor.js:105]: this should likely also be `styleTagOptions` (similarly for the script tag) given that's what they are..

Comment (abrom) [reply lib/grover/js/processor.js:105]: it'd possibly also be an array (which would be enumerated to allow multiple style/script tag options)

Comment (paresharma) [reply lib/grover/js/processor.js:105]: I used the names from the puppeteer doc, but agree to changing it to reflect what they really are. I'll add entries to README too.

> it'd possibly also be an array (which would be enumerated to allow multiple style/script tag options)

True, it is indeed an array. I'll update it.

**Comment by abrom** [comment]
Any update on this @paresharma  ?

**Comment by abrom** [comment]
Closing this due to inactivity. Feel free to re-open once you've addressed the feedback

**Pull request closed without merging**